src/main/resources/application.properties

```diff
@@ -1,7 +1,7 @@
 # Database connection properties
 spring.datasource.url=jdbc:postgresql://localhost:5432/test
-spring.datasource.username=developer
-spring.datasource.password=developer
+spring.datasource.username=admin
+spring.datasource.password=admin
```

Comment:
Does the team is aware of this change? The entire team should have the same credentials for the database user.